buildconfig/class_maker.py

```diff
@@ -18,6 +18,11 @@
 
 VERSION = "1.0"
 
+#======================================================================
+def getYear():
```

Comment:
```suggestion
def get_year():
```

We follow pep8 which [defines `snake_case` for function names](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names).

Reply:
Ah yes good catch!